openlibrary/plugins/openlibrary/home.py

```diff
@@ -32,25 +32,34 @@
     'preset:authorsalliance_mitpress': '(openlibrary_subject:(authorsalliance) OR collection:(mitpress) OR publisher:(MIT Press) OR openlibrary_subject:(mitpress)) AND (!loans__status__status:UNAVAILABLE)'
 }
 
+def get_homepage():
+    if 'env' not in web.ctx:
+        delegate.fakeload()
+    try:
+        stats = admin.get_stats()
+    except Exception:
+        logger.error("Error in getting stats", exc_info=True)
+        stats = None
+    blog_posts = get_blog_feeds()
+    page = render_template(
+        "home/index", stats=stats,
+        blog_posts=blog_posts,
+    )
+    page.v2 = True
+    return page
```

Comment:
@hornc perhaps the cache is having trouble serializing page to json and instead we should be returning page.dict() or dict(page)?